### DISCUSSION TITLE
Keycloak automatic provisioning of users/groups (via SCIM) to AWS IAM Identity Center

### DISCUSSION THREAD

**Discussion by jdhakar1995** [keycloak/keycloak#29444] [Q&A]
Hi Team,

We are using keycloak 20.0.3 as the IDP with **AWS IAM Identity Center(SSO).** We are using SCIM protocol to sync users and groups from keycloak to AWS SSO. But this sync does not work automatically. We have to invoke scim endpoint periodically to do the sync.

Does any keycloak version supports automatic provisioning of Users and Groups to **AWS SSO** using **SCIM**? If yes, please share any reference or documentation of how to configure it.

Thanks,
Jitendra

**Comment by jonkoops** [comment]
No, Keycloak currently does not support SCIM, support for this can be tracked under https://github.com/keycloak/keycloak/issues/13484. Please also note that the version of Keycloak you are running is no longer supported and contains several CVEs.

**Reply by jdhakar1995** [reply]
Thanks @jonkoops for your reply. I found this 3rd party SCIM Plugin- [keycloak-scim](https://github.com/mitodl/keycloak-scim). I tried to follow the steps but unfortunately it didn't work for me. Do you have any idea if this is the right SCIM Plugin to integrate keycloak with AWS IAM Identity Center for automatic provisioning of Users and Groups via SCIM?

**Reply by sachingaikwad123** [reply]
@jdhakar1995 Did that keycloak-scim plugin work? I am also evaluating how I can do user/groups provisioning of user/groups from my Django backend (MySQL) to keycloak server.
